src/System.Security.Cryptography.X509Certificates/tests/PublicKeyTests.cs

```diff
@@ -91,6 +91,16 @@ public static void TestPublicKey_Key_DSA()
             VerifyKey_DSA((DSA)alg);
         }
 
+        [Fact]
+        public static void TestGetDsaPublicKey()
```

Comment:
I see there is no test coverage for the private key side, either here or in underlying X509Certificate2.PrivateKey property with DSA. If you can add it great, otherwise I'll create a test issue to add it (which may need to include additional test data that contains a private key).